vips/vip-152-testnet.ts

```diff
@@ -0,0 +1,122 @@
+    againstDescription:
+      "I do not think that Venus Protocol should proceed with setting the vBNB admin and setting up PSR",
+    abstainDescription:
+      "I am indifferent to whether Venus Protocol proceeds with setting the vBNB admin and setting up PSR",
+  };
+
+  return makeProposal(
+    [
+      {
+        target: PROXY_ADMIN,
+        signature: "upgrade(address,address)",
+        params: [RISK_FUND, NEW_RISK_FUND_IMPLEMENTATION],
+      },
+      {
+        target: VBNBAdmin,
+        signature: "acceptOwnership()",
+        params: [],
+      },
+      {
+        target: vBNB_ADDRESS,
+        signature: "_setPendingAdmin(address)",
+        params: [VBNBAdmin],
+      },
+      {
+        target: VBNBAdmin,
+        signature: "_acceptAdmin()",
+        params: [],
+      },
+      {
+        target: PSR,
+        signature: "acceptOwnership()",
+        params: [],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [PSR, "addOrUpdateDistributionConfigs(DistributionConfig[])", NORMAL_TIMELOCK],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [PSR, "addOrUpdateDistributionConfigs(DistributionConfig[])", FAST_TRACK_TIMELOCK],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [PSR, "addOrUpdateDistributionConfigs(DistributionConfig[])", CRITICAL_TIMELOCK],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [PSR, "removeDistributionConfig(Schema,address)", CRITICAL_TIMELOCK],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [PSR, "removeDistributionConfig(Schema,address)", FAST_TRACK_TIMELOCK],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [PSR, "removeDistributionConfig(Schema,address)", NORMAL_TIMELOCK],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [PSR, "addOrRemoveAssetFromPrime(address,bool)", CRITICAL_TIMELOCK],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [PSR, "addOrRemoveAssetFromPrime(address,bool)", FAST_TRACK_TIMELOCK],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [PSR, "addOrRemoveAssetFromPrime(address,bool)", NORMAL_TIMELOCK],
+      },
+      {
+        target: PSR,
+        signature: "addOrUpdateDistributionConfigs((uint8,uint8,address)[])",
+        params: [
+          [
+            [0, 50, RISK_FUND],
+            [0, 50, TREASURY],
+            [1, 40, RISK_FUND],
+            [1, 60, TREASURY],
```

Comment:
I would change it here too, because I assume we'll remove VIP-153-testnet and only this file will be kept in the repo. So, it's better if we keep it completed and correct

```suggestion
            [0, 40, RISK_FUND],
            [0, 60, TREASURY],
            [1, 50, RISK_FUND],
            [1, 50, TREASURY],
```